feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -323,21 +323,26 @@ func TestZeroMaster(t *testing.T) {
 		highID:     inId0,
 		handle:     clientConnection(t, dut),
 		deviceID:   deviceId,
-		wantFail:   true,
-		wantStatus: 3,
+		wantWrite:  false,
```

Comment:
wantWrite and wantRead are unused. If they remain unused, please delete

Reply:
Thanks @greg-dennis. These variables not used anymore.